io.openems.edge.predictor.api/src/io/openems/edge/predictor/api/prediction/Prediction.java

```diff
@@ -146,6 +147,20 @@ public static Prediction from(ValueRange valueRange, ZonedDateTime time, Integer
 		return from(valueRange, buildMap(time, values));
 	}
 
+	/**
+	 * Constructs a {@link Prediction} object.
+	 * 
+	 * @param valueRange the {@link ValueRange}
+	 * @param interval   the Interval
+	 * @param time       the base time of the prediction values, rounded down to 15
+	 *                   minutes
+	 * @param values     the quarterly prediction values.
+	 * @return a {@link Prediction} object
+	 */
+	public static Prediction from(ValueRange valueRange, Interval interval, ZonedDateTime time, Integer... values) {
+		return from(valueRange, buildMapDuoDecimus(time, values));
```

Comment:
Need to discuss this "duodecimus" thing... it breaks a lot of assumptions (and Javadocs) if we suddenly have 5-minute predictions...